src/main/resources/sql/dummy-data/README.md

```
@@ -0,0 +1,186 @@
+
+### 데이터 확인
+```sql
+-- 생성된 데이터 확인
+SELECT COUNT(*) FROM member_tb;
+SELECT COUNT(*) FROM ai_model_tb;
+SELECT COUNT(*) FROM model_statistics_tb;
+```
+
+---
+
+**🎉 완벽하게 작동하는 모듈화된 구조로 NoModel 프로젝트의 모든 더미 데이터를 간편하게 생성하세요!**
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)